openmls/src/extensions/mod.rs

```diff
@@ -85,30 +88,37 @@ pub enum ExtensionType {
     /// Reserved. This must not be used.
     Reserved = 0,
```

Comment:
Can you drop this one? It should not be able to instantiate and OpenMLS moved towards not defining the reserved value.